### PR TITLE
Add LZ4 compression support to arm64 kernel

### PR DESCRIPTION
**Pull request by eccgecko**

### CODE REVIEW

**Comment by eccgecko** [comment]
Hoping to fix same issue referenced [here](https://github.com/raspberrypi/linux/issues/1875)

File: arch/arm64/configs/bcmrpi3_defconfig [hunk 1]
```diff
@@ -1272,6 +1272,7 @@ CONFIG_CRYPTO_TGR192=m
 CONFIG_CRYPTO_WP512=m
 CONFIG_CRYPTO_CAST5=m
 CONFIG_CRYPTO_DES=y
++CONFIG_CRYPTO_LZ4=m
```
Comment (pelwell) [inline arch/arm64/configs/bcmrpi3_defconfig]: This doesn't look right to me - have you included a `+` at the start of your added line?

Comment (eccgecko) [reply arch/arm64/configs/bcmrpi3_defconfig]: Yes. Sorry. It wasn't seeming to let me submit without it :/

**Comment by eccgecko** [comment]
Thanks. My first contribution to GitHub :)

**Pull request merged**